e2e/support/helpers/e2e-scrollbar-helpers.js

```diff
@@ -21,3 +21,40 @@ export const isScrollableVertically = element => {
 
   return isVerticalScrollbarVisible;
 };
+
+export const assertDescendantNotOverflowsContainer = (
+  descendant,
+  container,
+  message,
+) => {
+  const containerRect = container.getBoundingClientRect();
+  const descendantRect = descendant.getBoundingClientRect();
+
+  if (descendantRect.height === 0 || descendantRect.width === 0) {
+    return;
+  }
+
+  expect(descendantRect.bottom, `${message} bottom`).to.be.lte(
+    containerRect.bottom,
+  );
+  expect(descendantRect.top, `${message} top`).to.be.gte(containerRect.top);
+  expect(descendantRect.left, `${message} left`).to.be.gte(containerRect.left);
+  expect(descendantRect.right, `${message} right`).to.be.lte(
+    containerRect.right,
+  );
+};
+
+export const assertIsEllipsified = element => {
+  expect(isEllipsified(element), "is ellipsified").to.equal(true);
+};
+
+export const assertIsNotEllipsified = element => {
+  expect(isEllipsified(element), "is ellipsified").to.equal(false);
+};
+
+export const isEllipsified = element => {
+  return (
+    element.scrollHeight > element.clientHeight ||
+    element.scrollWidth > element.clientWidth
+  );
+};
```

Comment:
Not sure if these 3 functions belong in a file called `e2e-scrollbar-helpers`.

Maybe let's rename the file to `e2e-overflow-helpers`?